File: Modul6/Soal4/PRAK504-2210817110008-Aditya Oktaviari.py
def reverse(A):
    rev=0
    while(A):
        rev=rev*10
        rev=rev+A%10
        A //=10
    return rev

A,B=input().split()
A=int(A)
B=int(B)
A=reverse(A)
B=reverse(B)
C = A+B
print(reverse(C))

File: Modul6/Soal4/PRAK504-2210817110008-Aditya Oktaviari.c
#include <stdio.h>
int reverse(int A){
    int rev=0;
    while(A!=0){
        rev=rev*10;
        rev=rev+A%10;
        A= A/10;
    }
    return rev;
}


int main() {
int A, B;
scanf("%d %d",&A,&B);
A=reverse(A);
B=reverse(B);
int C = A+B;
printf("%d",reverse(C));
}
